core/utils_test.py

```diff
@@ -75,7 +80,7 @@ def test_yaml_dict_conversion(self) -> None:
             utils.InvalidInputException,
             'while parsing a flow node\n'
             'expected the node content, but found \'<stream end>\'\n'):
-            yaml_str = utils.dict_from_yaml('{')
+            yaml_str = utils.dict_from_yaml('{') #type: ignore
```

Comment:
Why is the ignore needed?

Reply:
since we expect a Dict[str, Any]  in the function definition hence the single braces was throwing an error for this case

Reply:
it is expecting a string as argument and it is expecting Dict[str, Any] for return type.
It should not fail here, can u post log.

Reply:
Starting Mypy type checks.
core/utils_test.py:83: error: Incompatible types in assignment (expression has type "Dict[str, Any]", variable has type "str")  [assignment]
Found 1 error in 1 file (checked 1 source file)